benchmark/src/bench.rs

```diff
@@ -18,13 +18,22 @@ use grpc_proto::testing::services_grpc::BenchmarkService;
 use grpc_proto::util;
 use grpcio::GrpcSlice;
 
+#[cfg(feature = "protobuf-codec")]
 fn gen_resp(req: &SimpleRequest) -> SimpleResponse {
     let payload = util::new_payload(req.get_response_size() as usize);
     let mut resp = SimpleResponse::default();
     resp.set_payload(payload);
     resp
 }
 
+#[cfg(feature = "protobufv3-codec")]
```

Comment:
Seems protobuf v2 can also use this logic as all fields in v2 are public. And there are also other places in this PR can use the same code in both v2 and v3.

Reply:
Good catch, this makes the code simpler. It doesn't always work due to type differences, but I went over the changes and refactored what I could. Thank you for the review!